.gitmodules

```diff
@@ -6,7 +6,7 @@
 	url = https://github.com/heroku/cli-engine-example
 [submodule "cli-engine-test-foo"]
 	path = cli-engine-test-foo
-	url = git@github.com:jdxcode/cli-engine-test-foo.git
+	url = https://github.com/jdxcode/cli-engine-test-foo.git
```

Comment:
this is to make appveyor happy, which sadly didn't fix the appveyor build, but it does run further